_posts/2022/2022-01-26-dev-experience-2022.md

```diff
@@ -0,0 +1,89 @@
+
+A few days later, I got a call for interview. The interview was conducted in 2 parts. First one was basic problem solving, and we also discussed internet protocols and basic web vulnerabilities.
+
+The 2nd round was based on my interests, how I would tackle a real world problem and any questions I would like to ask SDSLabs. I was asked to create the Tic-Tac-Toe game and a library management system. This was also a fanboy moment for me as Twarit Waikar was one of the interviewers (SDSLabs 2021 alumnus). He is a game developer and worked extensively on [Rubeus](https://github.com/sdslabs/Rubeus) and [Rootex](https://github.com/sdslabs/Rootex) game engines. I had read his blogs and looked at his games and the game engines and was pretty impressed by that (*Making a game engine seemed a very huge undertaking to me*). I asked them about game engines and game development and web security.
+
+## The Second Semester
+
+#### The initiation
+
+I got a phone call informing me about my induction to SDSLabs. We had an intro chapo (online, coz covid) and got to know everyone there. We were added on the workspace apps and this got the cart rolling.
+
+The semester was filled with lectures on various subjects (Git, Bash scripting, Web development, Game development) and we has follow up assignments. The follow up assignments spurred me to learn about something new and implement it within a deadline.
+
+After this, we were explained about all the apps and projects going on in SDSLabs. We were briefed about their status, what technology they are built upon, what problems they solve and what more work needs to be done.
+
+#### Minors!!
+
+The day came when we were asked to pick up a project from the ongoing ones to complete a minor issue in them. This was sorting-hat-esque where everyone looked at that spreadsheet of projects and open issues and put their name against any one of them. Minors were completed in less than a week.
+
+## The Third Semester
+
+#### Hackathons
+
+By now we started participating in various hackathons (*and managing to win some of them!!*). Members would come together, form teams among themselves (sometimes even our alumni joined us) and we would participate in the hackathons. Sitting through hours of code sprints and preparing POCs, those were exhilarating hours.
+
+#### Majors
+After the minors, we had to choose our Majors: solving a large issue or implementing something important in the ongoing projects. We again met for the sorting and were assigned the responsibilities.
+
+#### Releasing a Project
+
+When a project comes to fruition and is ready to be open-sourced, everyone at SDSLabs is pretty excited about it. Seeing the hardwork of our members getting released ittruly a moment to be cherished. I witnessed the release of [ZapDB](https://github.com/sdslabs/zap-db) and [Studyportal](https://github.com/sdslabs/studyportal).
+
+#### Information Security in SDSLabs
```

Comment:
A separate section for security seems a little unnecessary. If the point is to illustrate labs branching out into non-dev fields, highlighting one of said fields presents said diversification through a very restricted lens by undermining other areas of tech that our members have explored/are exploring. BackdoorCTF toh uda de imo but if you wanna write about your experience playing CSAW in a team with other labs folk or highlight your achievements, I would suggest adjusting that elsewhere and then merge this into the next section, expanding upon it by mentioning people around you working in areas which aren't strictly de. You can bring up ML, research, hardware, CS theory e.t.c in addition to security so it feels more inclusive (interest wise) (You cold also use your CTF experience as an example of exploration in this section as opposed to being the highlight of it)